cli/src/spec/SpecCommand.py

```diff
@@ -58,5 +58,5 @@ def get_spec_groups():
         for entry in listdir:
             if os.path.isdir(f'{SPEC_TEST_PATH}/spec/{entry}'):
                 groups = groups + [entry]
-        sorted(groups, key=str.lower)
-        return groups
+        sorted_group = sorted(groups, key=str.lower)
+        return sorted_group
```

Comment:
Just return sorted()? I don't think there is a need for the sorted_group variable.

Reply:
Good point:)